setup.cfg

```diff
@@ -160,6 +160,7 @@ install_requires =
     typing-extensions>=3.7.4;python_version<"3.8"
     unicodecsv>=0.14.1
     werkzeug~=1.0, >=1.0.1
+    pypiserver~=1.4
```

Comment:
We only need pypi server for development. There is no need to make whole airflow depend on it in `install_requires`. Please move it to `[devel]` extra in `setup.py`.